app/app.pro

```diff
@@ -164,7 +164,7 @@ win32 {
     RC_FILE = data/pencil2d.rc
 }
 
-linux {
+unix {
```

Comment:
I'm fairly certain this should be
```suggestion
unix:!macx
```

Because Mac OS uses other conventions which are setup above.

Reply:
@scribblemaniac I thought so too, but scribblemaniac suggested doing it this way, see https://github.com/pencil2d/pencil/issues/1127#issuecomment-440585025.

Reply:
Hmm.. ah maybe that's true... alright then, no further comments. Looks fine to me.